src/lib.rs

```diff
@@ -190,6 +195,12 @@ pub use crate::sse42::deser::*;
     target_feature = "neon"
 ))))]
 use crate::sse42::stage1::{SimdInput, SIMDINPUT_LENGTH, SIMDJSON_PADDING};
+#[cfg(all(not(any(
+    target_feature = "sse4.2",
+    target_feature = "avx2",
+    target_feature = "neon"
+))))]
```

Comment:
I think this guard is slighty off
```suggestion
#[cfg(all(
    target_feature = "sse4.2",
    not(any(
      target_feature = "avx2",
      target_feature = "neon"
))))]
```

Reply:
I copied that from the preceding lines. I assume that is meant to be the fallback case?

Reply:
My apologies you're right :) I didn't unfold the previous lines!